components/PaginationPage/index.tsx

```diff
@@ -0,0 +1,80 @@
+interface Props {
```

Comment:
question about the filename for this - it's not really a Page is it? Like its a component? Just Pagination would do right? Or am I missing something

Reply:
Ah yes, just a component, a leftover from a previous iteration! Renamed to just `Pagination`